lib/ex_machina.ex

```diff
@@ -150,7 +141,7 @@ defmodule ExMachina do
 
   ## Example
 
-      factory :user do
+      def factory :user, _attrs do
```

Comment:
I think we should add parens around the parameters. I think that's a common convention